### PR TITLE
align to recent electron api 

### PR DESCRIPTION
**Pull request by andirotter**
plus some minor clean up and refactoring, feel free to merge

### CODE REVIEW

**Comment by pwambach** [comment]
Hey, great thanks! I will look into it in more detail in the next days. One question which came to my mind right away... I used q as a Promise lib because at thus time I was quite into Angular. Cool that you refactores this! But do we need a Promise lib at all? Why not use the native Promise implementation?

**Comment by andirotter** [comment]
Hey, thanks for the reply, and of course you're right, for this case it will work fine with the native Promise implementation. through my work with bluebird the last two years i'm - just like you with q - totally into this and forgetting always, that we have now the full native Promises in nodejs. will update this.

File: README.md [hunk 1]
```diff
@@ -12,5 +12,12 @@ For performance reasons FatFileFinder currently ignores files smaller than 1 MB
 
 [FatFileFinder-0.1.0.zip](https://github.com/pwambach/fat-file-finder/releases/download/0.1.0/FatFileFinder-0.1.0.zip)
 
+# Build it on your own
+
+1. ´´´git clone ttps://github.com/pwambach/fat-file-finder.git && cd fat-file-finder´´
```
Comment (IMcPwn) [inline README.md]: I think you mean **h**ttps://github.com/pwambach/fat-file-finder.git

Comment (andirotter) [inline README.md]: pssss, its my prototype for  supersecret next generation data transport protocol ;-). of course, yes

**Comment by pwambach** [comment]
Thanks again. Really appreciate your effort to keep this up to date!

**Comment by pwambach** [comment]
FYI: I added a `npm start` script which runs webpack and electron directly and without the need to install electron-prebuilt globally. Ran into this today because I'm not on the same laptop I used to have when I wrote this app :)

**Comment by andirotter** [comment]
nice and no thanks for that, was a good introduction into building electron apps, will add this too

**Pull request merged**